src/settings/MappingProfiles/detailsSections/edit/InvoiceDetailSection/InvoiceLineAdjustments.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  useIntl,
+  FormattedMessage,
+} from 'react-intl';
+import { Field } from 'redux-form';
+
+import {
+  Accordion,
+  Row,
+  Col,
+  Card,
+  IconButton,
+  RepeatableField,
+  TextField,
+  Checkbox,
+} from '@folio/stripes/components';
+
+import { TypeToggle } from '@folio/stripes-acq-components';
+
+import { AcceptedValuesField } from '../../../../../components';
+
+import {
+  onAdd,
+  onRemove,
+  getSubfieldName,
+} from '../../utils';
+import { TRANSLATION_ID_PREFIX } from '../../constants';
+import {
+  createOptionsList,
+  mappingProfileSubfieldShape,
+  okapiShape,
+  INOVOICE_ADJUSTMENTS_RELATION_TO_TOTAL_OPTIONS,
+} from '../../../../../utils';
+
+export const InvoiceLineAdjustments = ({
+  lineAdjustments,
+  initialFields,
+  setReferenceTables,
+  okapi,
+}) => {
+  const { formatMessage } = useIntl();
+
+  const relationToTotalList = createOptionsList(INOVOICE_ADJUSTMENTS_RELATION_TO_TOTAL_OPTIONS, formatMessage);
+
+  const renderLineAdjustment = (field, index) => {
+    return (
+      <Card
+        headerEnd={(
+          <IconButton
+            data-test-repeatable-field-remove-item-button
+            icon="trash"
+            onClick={() => onRemove(index, lineAdjustments, 41, setReferenceTables, 'order')}
+            size="medium"
+            ariaLabel={formatMessage({ id: 'stripes-components.deleteThisItem' })}
+          />
+        )}
+        headerStart={(
+          <FormattedMessage
+            id={`${TRANSLATION_ID_PREFIX}.invoice.invoiceAdjustments.adjustments.sectionTitle`}
+            values={{ index: index + 1 }}
+          />
+        )}
+      >
+        <Row left="xs">
+          <Col xs={3}>
+            <Field
+              component={TextField}
+              label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.invoice.invoiceAdjustments.field.description`} />}
+              name={getSubfieldName(41, 0, index)}
+            />
+          </Col>
+          <Col xs={2}>
+            <Field
+              component={TextField}
+              label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.invoice.invoiceAdjustments.field.amount`} />}
+              name={getSubfieldName(41, 1, index)}
+            />
+          </Col>
+          <Col xs={2}>
+            <Field
+              component={TypeToggle}
+              label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.invoice.invoiceAdjustments.field.type`} />}
+              name={getSubfieldName(41, 2, index)}
+              currency={undefined}
```

Comment:
Do you really need to pass `undefined` to this prop?

Reply:
done